lib/cisco_node_utils/cmd_ref/interface_portchannel.yaml

```diff
@@ -28,6 +28,8 @@ lacp_max_bundle:
   set_value: "lacp max-bundle %s"
   N3k:
```

Comment:
Better to start using yaml anchor and aliases here.

https://github.com/cisco/cisco-network-node-utils/blob/develop/lib/cisco_node_utils/cmd_ref/README_YAML.md#yaml-anchors-and-aliases